core/WFCORE-4758_Simple_config_export_for_a_server_as_an_attachment_for_standalone_or_domain_mode.adoc

```diff
@@ -0,0 +1,107 @@
+= Simple config export for a server as an attachment for standalone or domain mode 
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-4758[WFCORE-4758]
+* https://issues.redhat.com/browse/EAP7-1810[EAP7-1810]
+
+=== Related Issues
+
+* N/A
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* HAL
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+Provide a management operation to export the configuration of a running server as an XML data stream to avoid content being escaped. This operation should replace the existing '*read-config-as-xml*' operation without the escaping issues.
+It should work for a standalone server as for a domain server.
+
+=== Hard Requirements
+
+ * Export the current running configuration as an XML data stream that can be saved or displayed with the attachment operation from the CLI.
```

Comment:
Please add a requirement that the operation should fail if the user is not in a role that allows reading sensitive data.